jenkins-master/jobs/mozilla-central_update/config.xml

```diff
@@ -103,20 +80,22 @@ NSPR_LOG_FILE=http.log</propertiesContent>
       <project>scripts</project>
       <filter></filter>
       <target></target>
-      <selector class="hudson.plugins.copyartifact.StatusBuildSelector"/>
+      <selector class="hudson.plugins.copyartifact.StatusBuildSelector">
+        <stable>true</stable>
+      </selector>
     </hudson.plugins.copyartifact.CopyArtifact>
     <hudson.plugins.xshell.XShellBuilder plugin="xshell@0.9-SNAPSHOT">
       <commandLine>python -u submission.py --test-type=update --build-state=running --repository=mozilla-central --revision=$REVISION --locale=$LOCALE --update-number=$UPDATE_NUMBER treeherder_venv</commandLine>
       <executeFromWorkingDir>false</executeFromWorkingDir>
     </hudson.plugins.xshell.XShellBuilder>
     <hudson.plugins.xshell.XShellBuilder plugin="xshell@0.9-SNAPSHOT">
-      <commandLine>python runtests.py --type=update --installer-url=$INSTALLER_URL --update-channel=$CHANNEL --update-target-build-id=$TARGET_BUILD_ID</commandLine>
+      <commandLine>python -u runtests.py --test-type=update --repository=mozilla-central --installer-url=$INSTALLER_URL --update-channel=$CHANNEL --update-target-build-id=$TARGET_BUILD_ID</commandLine>
       <executeFromWorkingDir>false</executeFromWorkingDir>
     </hudson.plugins.xshell.XShellBuilder>
   </builders>
   <publishers>
     <hudson.tasks.ArtifactArchiver>
-      <artifacts>build/upload/*.*</artifacts>
+      <artifacts>build/http.log,build/upload/**</artifacts>
```

Comment:
So I cannot move the `http.log` to `build/upload/` because a possible clobber step in mozharness could purge this file then. We don't have control about that from the CI level.